library/src/main/java/com/novoda/downloadmanager/VersionOnePartialDownloadBatch.java

```diff
@@ -0,0 +1,54 @@
+package com.novoda.downloadmanager;
+
+import java.util.List;
+
+public class VersionOnePartialDownloadBatch {
```

Comment:
`Partials` only needed a subset of the data that a `Migration` captures so created it's own dedicated object.